pkg/cloudprovider/aws/instance.go

```diff
@@ -36,11 +40,30 @@ import (
 	"github.com/awslabs/karpenter/pkg/utils/options"
 )
 
+const (
+	iceCacheODTTL           = 15 * time.Second
```

Comment:
```suggestion
	InsufficientCapacityExceptionCacheOnDemandTTL           = 15 * time.Second
```

Reply:
I think the spot and OD cache TTLs should be the same 45 seconds.